.github/workflows/test_python_ver_matrix.yml

```diff
@@ -0,0 +1,65 @@
+name: Python tests / Ubuntu
+on:
+  push:
+    branches:
+      - develop
+      - master
+
+  pull_request:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build:
+    name: python version matrix
+
+    runs-on: ubuntu-latest
+    
+    env:
+      AMICI_SKIP_CMAKE_TESTS: "TRUE"
+    
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - run: echo "::set-env name=AMICI_DIR::$(pwd)"
+    - run: echo "::set-env name=BNGPATH::${AMICI_DIR}/ThirdParty/BioNetGen-2.3.2"
+    
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 20
+
+    # install dependencies
+    - name: apt
+      run: |
+        sudo apt-get update \
+          && sudo apt-get install -y \
+          swig \
+          libatlas-base-dev \
+          libhdf5-serial-dev
+    - name: pip
+      run: |
+        pip3 install --upgrade --user wheel \
+          && pip3 install --upgrade --user setuptools
+          
+
+    # install AMICI
+    - name: Build BNGL
+      run: |
+        scripts/buildBNGL.sh
+    - name: Install python package
+      run: |
+        scripts/installAmiciSource.sh
```

Comment:
Maybe use sdist here, as that is the way how most users will use it?

Reply:
Had that initially, but having notebooks run outside a virtualenv was a major pain